Packages/com.unity.inputsystem/InputSystem/Actions/InputActionSetupExtensions.cs

```diff
@@ -225,6 +225,49 @@ private static int AddBindingInternal(InputActionMap map, InputBinding binding)
             return bindingIndex;
         }
 
+        public static BindingSyntax ChangeBinding(this InputAction action, int index)
```

Comment:
What's the use case for using indexes? Iterating over all bindings?

Reply:
Stuff like

```
for (var i = 0; i < myAction.bindings.Count; ++i)
    if (myAction.bindings[i].SomeKindOfCheck)
        myAction.ChangeBinding(i)....;
```